salt/cli/standalone_proxy.py

```diff
@@ -0,0 +1,448 @@
+# -*- coding: utf-8 -*-
```

Comment:
Maybe it makes more sense to add these to the `salt` cli script...

Reply:
This just looks like a copy of the regular minion cli module?